packages/bruno-app/src/components/ResponsePane/QueryResult/StyledWrapper.js

```diff
@@ -3,7 +3,8 @@ import styled from 'styled-components';
 const StyledWrapper = styled.div`
   display: grid;
   grid-template-columns: 100%;
-  grid-template-rows: 1.25rem calc(100% - 1.25rem);
+  grid-template-rows: ${(props) =>
+    props.queryFilterEnabled ? '1.25rem calc(100% - 3.5rem)' : '1.25rem calc(100% - 1.25rem)'};
```

Comment:
```suggestion
    props.queryFilterEnabled ? '1.25rem 1fr 2.25rem' : '1.25rem 1fr'};
```
You can utilize the CSS-Grid here and add grid row for your input. I also suggest to change `calc(100% - 1.25rem)` to `1fr` to make it cleaner.